test/box/net.box_msgpack_gh-2195.test.lua

```diff
@@ -13,10 +13,6 @@ c:eval('return echo(...)', {42})
 c:call('echo', 42)
 c:eval('return echo(...)', 42)
 c:close()
-c = net.connect(box.cfg.listen, {call_16 = true})
-c:call('echo', 42)
-c:eval('return echo(...)', 42)
-c:close()
```

Comment:
Nit: You need to drop more than that - up to `-- CALL vs CALL_16 in connect options`.

Reply:
Why should I drop the part testing regular `call`?

Reply:
Applied your suggestion.